backend/src/routes/api/modelRegistries/index.ts

```diff
@@ -0,0 +1,141 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { PatchUtils } from '@kubernetes/client-node';
+import { secureAdminRoute } from '../../../utils/route-security';
+import { KubeFastifyInstance, ModelRegistryKind, RecursivePartial } from '../../../types';
+import { MODEL_REGISTRY_NAMESPACE } from '../../../utils/constants';
+
+export default async (fastify: KubeFastifyInstance): Promise<void> => {
```

Comment:
Ok, first thing, checking this, we need to add permissions to the Dashboard Serviceaccount to make sure it can fetch Model Registry CR. I would advise to add that to the [cluster role file](https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/base/cluster-role.yaml)

Reply:
Done, pushed -- I gave it all the verbs (same as we do for notebooks, for example), is that right?

Reply:
Yes, that's fine cause we need to do all the operations 👌